connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -462,11 +465,20 @@ private void expectAdd(String name,
         // And we should instantiate the tasks. For a sink task, we should see added properties for
         // the input topic partitions
         Map<String, String> generatedTaskProps = taskConfig(taskClass, sink);
-        EasyMock.expect(worker.connectorTaskConfigs(CONNECTOR_NAME, DEFAULT_MAX_TASKS, TOPICS_LIST))
+
+        if (sink) {
```

Comment:
You could just use a ternary operator to select what the topic list parameter will be and avoid duplicating the code in the two blocks here.